spec/unit/interactive-auth.spec.ts

```diff
@@ -174,9 +343,108 @@ describe("InteractiveAuth", function() {
             throw err;
         });
 
-        return ia.attemptAuth().catch(function(error) {
-            expect(error.message).toBe('No appropriate authentication flow found');
```

Comment:
This was not actually asserted since it was never asserted that the `catch` happens in the first place. In fact it did not but does now.